CoreVisualGraph/src/au/gov/asd/tac/constellation/graph/visual/utilities/VisualGraphUtilities.java

```diff
@@ -276,4 +278,24 @@ public Vector3f next() {
             return new Vector3f(screenLocation[0], screenLocation[1], screenLocation[2]);
         });
     }
+    
+        public static List<Integer> getSelectedElements(final GraphReadMethods graph) {
+        List<Integer> selectedIds = new ArrayList<>();
+        final int vertexSelectedAttribute = VisualConcept.VertexAttribute.SELECTED.get(graph);
+        if (vertexSelectedAttribute != Graph.NOT_FOUND) {
+            graph.vertexStream().forEach(vertexId -> {
+                if (graph.getBooleanValue(vertexSelectedAttribute, vertexId)) {
+                    selectedIds.add(vertexId);
+                }
+            });
+        
+            
+//            final GraphIndexResult result = grm.getElementsWithAttributeValue(vertexSelectedAttribute, Boolean.TRUE);
+//            final int resultCount = result.getCount();
+//            for (int i = 0; i < resultCount; i++) {
+//                selectedIds.add(result.getNextElement());
+//            }
```

Comment:
can this be removed?

Reply:
It can, but instead I have left it in and added a comment explaining that it is a suggested improvement pending an investigation into the indexing with Constellation